spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -188,4 +210,17 @@ def it_redirects_to_user_login
       end
     end
   end
+
+  describe "show" do
+    let(:chaptered_work) { create(:work, title: "Cool title") }
+    let(:chapter2) { create(:chapter, work: chaptered_work, position: 2, posted: true, title: "Second title") }
+    let(:bookmark) { create(:bookmark, bookmarkable_id: chaptered_work.id) }
+
+    it "shows a bookmark" do
```

Comment:
How about `it "returns bookmark"` to go with the phrasing used elsewhere in this file? It would probably also make sense to give this some context descriptions to make it easier for people to slot in additional tests in the future, e.g.:
```
context "when logged in as owner of bookmarker" do
  context "when bookmark is public" do
    context "when bookmarkable is chaptered public work" do
      it "returns bookmark" do
        etc
      end
    end
    
    context "when bookmarkable is work hidden by admin" do
      it "does something" do
        etc
      end
    end
  end

  context "when bookmark is private" do
    context "when bookmarkable is a restricted series" do
      it "does something" do
        etc
      end
    end
  end
end

context "when logged in as admin" do
  context "when bookmark is private" do
    context "when bookmarkable is a restricted series" do
      it "does something" do
        etc
      end
    end
  end
end
```

Reply:
Good idea, and I might add some more of those later.